src/config.hpp

```diff
@@ -107,6 +117,17 @@ class Config {
         return result->operator[]("rest_port").as<uint64_t>();
     }
 
+    /**
+         * @brief Get the rest network interface address to bind to
+         * 
+         * @return const std::string&
+         */
+    const std::string restBindAddress() {
+        if (result->count("rest_bind_address"))
+            return result->operator[]("rest_bind_address").as<std::string>();
```

Comment:
There is no sanity check of values coming for (rest|grpc)_bind_addresses.

Reply:
There is no test to document what will happen as well with strange values.

Reply:
added ip/domain validation